packages/office-addin-debugging/src/start.ts

```diff
@@ -60,7 +61,17 @@ export async function isPackagerRunning(statusUrl: string): Promise<boolean> {
 
 export async function isUrlAvailable(url: string): Promise<boolean> {
     try {
-        await fetch.default(url);
+        const serverOptions = await devCerts.getHttpsServerOptions();
+        const options: any = {
+            cert: serverOptions.cert,
+            key: serverOptions.key,
+            method: "GET",
+        };
+        options.agent = new https.Agent(options);
+        const request = https.request(url, options);
+        request.on("error", () => {
```

Comment:
This doesn't look right to me. It will immediately return true before the async "error" event would have a chance to return false. It would seem that for both the success and error case, there needs to be an async wait to know if the request was successful or not.

Reply:
I actually tested to ensure it was hitting the request.on(error), which it does the first couple times around because the url isn't available yet, so it return false the first couple times

Reply:
But I will look into it some more tomorrow

Reply:
On second thought I think you may be right. Will investigate more tomorrow. I’d like to get a fix in soon so we can update the templates with all the office-addin-scripts changes. Honestly I’m not even sure we need isUrlAvailable. This was put in originally to address test failures which have since been obviated by providing a dev port when calling startDebugging. That said it’d be nice to get it working

Reply:
@akrantz I think this is now working as expected.  I tested the latest implementation with Office-Addin-Taskpane